app/src/main/java/com/epam/ThirdFragmentNavigationDrawer.java

```diff
@@ -0,0 +1,21 @@
+package com.epam;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.epam.cleancodetest.R;
+
+public class ThirdFragmentNavigationDrawer extends Fragment {
```

Comment:
it will be more elegant to create common fragment and passing layout ID into args for this common fragment. This way will remove 2 from 3 fragments